pkg/deploy/controller/deployment/controller_test.go

```diff
@@ -20,28 +23,16 @@ import (
 // TestHandle_createPodOk ensures that a the deployer pod created in response
 // to a new deployment is valid.
 func TestHandle_createPodOk(t *testing.T) {
-	var (
-		updatedDeployment *kapi.ReplicationController
-		createdPod        *kapi.Pod
-		expectedContainer = okContainer()
-	)
+	expectedContainer := okContainer()
```

Comment:
> This doesn't seem to be used with the fake?

I was playing with factorizations.  I think I'm going to update `testclient` to accept a map of of reaction functions organized by verb/resource tuples along with plumbing the default return through.  It will play much nicer.